src/main/java/core/basesyntax/dao/ReadDataFromFile.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.dao;
```

Comment:
why dao? dao and service are different things, create separate packages for them
```suggestion
package core.basesyntax.service;
```